src/Label.js

```diff
@@ -1,24 +1,30 @@
 import styled from 'styled-components'
-import { space, fontSize, color, propTypes } from 'styled-system'
```

Comment:
Why importing `propTypes` from  `styled-system`? why not `prop-types`?

Reply:
This `propTypes` is one provided by styled-system that contain the prop-types for several of the styled-system components so you can append them to the component's prop types. See line 14-18